### PR TITLE
EZP-31007: ezplatform-kernel instead of ezpublish-kernel

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-31007
| Versions      | master (3.0)

Note: Merge after `v1.0.0` tag is created in `ezplatform-kernel`

### CODE REVIEW

File: docs/api/general_rest_usage.md [hunk 1]
```diff
@@ -44,10 +44,10 @@ Content-Type: application/vnd.ez.api.ContentInfo+xml
 ###### HTTP code
 
 The API responded here with a standard `200 OK` HTTP response code, which is the expected response code for a typical GET request.
-Some GET requests, like [getting a Content item's current version,](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#get-current-version) may return a `301 Moved permanently` or `307 Temporary redirect` code.
+Some GET requests, like [getting a Content item's current version,](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst#get-current-version) may return a `301 Moved permanently` or `307 Temporary redirect` code.
 
 Errors are indicated with HTTP error codes, e.g. `404 Not Found` or `500 Internal Server Error`.
-The [REST specifications](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst) provide the list of every HTTP response code you can expect from implemented resources.
+The [REST specifications](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst) provide the list of every HTTP response code you can expect from implemented resources.
```
Comment (MagdalenaZuba) [inline docs/api/general_rest_usage.md]:
```suggestion
The [REST specifications](https://doc.ezplatform.com/rest-api-reference) provide the list of every HTTP response code you can expect from implemented resources.
```

File: docs/api/general_rest_usage.md [hunk 2]
```diff
@@ -58,21 +58,21 @@ As long as a response contains an actual HTTP body, the Content Type header will
...
+As the example above shows, sending a PATCH `/content/objects/23` request with a [ContentUpdateStruct](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/API/Repository/Values/Content/ContentUpdateStruct.php) XML payload will update this content.
 
 REST will use the `Accept-Patch` header to indicate how to **modify** the returned **data**.
 
 ###### Location header
 
 Depending on the resource, request and response headers will vary.
 
-For instance [creating Content](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#creating-content) and [getting a Content item's current version](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#get-current-version)
+For instance [creating Content](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst#creating-content) and [getting a Content item's current version](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst#get-current-version)
```
Comment (MagdalenaZuba) [inline docs/api/general_rest_usage.md]:
```suggestion
For instance [creating Content](https://ezsystems.github.io/ezplatform-rest-reference/#managing-content-create-content-type) and [getting a Content item's current version](https://ezsystems.github.io/ezplatform-rest-reference/#managing-content-get-current-version)
```

File: docs/api/general_rest_usage.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ Location is sent by `201 Created`, `301 Moved permanently`, `307 Temporary redir
 
 This request header is the request counterpart of the Location response header.
 It is used for a COPY or MOVE operation on a resource to indicate where the resource should be moved to by using the ID of the destination.
-An example of such a request is [copying a Content item](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#copy-content).
+An example of such a request is [copying a Content item](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst#copy-content).
```
Comment (MagdalenaZuba) [inline docs/api/general_rest_usage.md]:
```suggestion
An example of such a request is [copying a Content item](https://ezsystems.github.io/ezplatform-rest-reference/#managing-content-copy-content).
```

File: docs/api/general_rest_usage.md [hunk 4]
```diff
@@ -363,7 +363,7 @@ Content-Type: application/vnd.ez.api.Session+xml
 The `csrfToken` is returned in the login response.
 It is important to keep the CSRF Token for the duration of the session as it needs to be sent with requests other than GET/HEAD when auth is set to session (in most cases it is).
 
-For details, see [Session-based authentication](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#session-based-authentication) in the REST specifications.
+For details, see [Session-based authentication](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst#session-based-authentication) in the REST specifications.
```
Comment (MagdalenaZuba) [inline docs/api/general_rest_usage.md]: This part is not covered in new REST Reference

File: docs/api/extending_the_rest_api.md [hunk 5]
```diff
@@ -292,7 +292,7 @@ myRestBundle_hello_world_using_post:
 
 !!! note
 
-    POST requests are not able to access the Repository without performing a user authentication. For more information check [REST API Authentication](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#authentication).
+    POST requests are not able to access the Repository without performing a user authentication. For more information check [REST API Authentication](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rest/REST-API-V2.rst#authentication).
```
[the remark this answers is not in the document]
Comment (alongosz) [reply docs/api/extending_the_rest_api.md]: specifications were dropped from kernel, so this link is not gonna work anyway

File: docs/api/field_type_reference.md [hunk 6]
```diff
@@ -2216,7 +2216,7 @@ When creating RichText content with the REST API, it is possible to provide data
 
 When the value given over REST API is transformed into a Field Type's `Value` object, it will be treated as a string. This means you can use any supported input format for input over REST API.
 
-For further information about the [internal implementation of RichText Field Type, see the kernel documentation](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rich_text/ezdocbook.md)
+For further information about the [internal implementation of RichText Field Type, see the kernel documentation](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/doc/specifications/rich_text/ezdocbook.md)
```
Comment (alongosz) [inline docs/api/field_type_reference.md]: dropped

File: docs/api/general_rest_usage.md [hunk 7]
```diff
@@ -129,7 +129,7 @@ Depending on how much data you need, you may choose to crawl those relations or
 
 ##### XSD files
 
-For each XML structure known to the REST API, you can find [XSD files](https://github.com/ezsystems/ezpublish-kernel/tree/master/doc/specifications/rest/xsd) in the XSD folder of the specifications.
+For each XML structure known to the REST API, you can find [XSD files](https://github.com/ezsystems/ezplatform-kernel/tree/master/doc/specifications/rest/xsd) in the XSD folder of the specifications.
```
Comment (alongosz) [inline docs/api/general_rest_usage.md]: dropped

File: docs/api/general_rest_usage.md (hunk 4 above)
Comment (alongosz) [reply docs/api/general_rest_usage.md]: dropped

**Comment by DominikaK** [comment]
@alongosz true, restored links to ezpublish-kernel for specifications, until the relevant parts are moved to developer-doc.
I'll recheck all links anywa after ezplatform-kernel v1.0.0 is tagged

**Pull request merged**